processing_handler.go

```diff
@@ -186,3 +194,11 @@ func handleProcessing(reqID string, rw http.ResponseWriter, r *http.Request) {
 
 	respondWithImage(ctx, reqID, r, rw, imageData)
 }
+
+func loadFallback() (err error) {
+	fallback, err = getFallbackData()
+	if err != nil {
+		logError("Could not load fallback data. Fallback images will not be available: %s", err.Error())
+	}
+	return
+}
```

Comment:
`getFallbackData` error should not be suppressed. Suppressing the error would introduce an unpredictable behavior.